### DISCUSSION TITLE
Copy/Paste Records from Pocketbase Collection into Excel/Sheets

### DISCUSSION THREAD

**Discussion by jspann93** [pocketbase/pocketbase#2652] [Q&A] [answered]
Hi all!

I've been using Pocketbase to store user-collected data for a custom poll. I want to be able to copy/paste the data from Pocketbase into Excel or Google Sheets for some basic analysis but I'm having trouble. I have no idea why but sometimes I can successfully move the data by highlighting all the Records and using a standard copy/paste (Control+C then Control+V) but other times that doesn't work at all. Am I missing something obvious? Is there an easy way to move records into a spreadsheet format? Thanks!!

**Comment by ganigeorgiev** [comment, marked as answer]
I don't think there is an easy way to copy data into an spreadsheet format at the moment.
This eventually will be handled as part of #48 (before that we'll need bulk/transactional web apis support which are planned for v0.18/v0.19).

**Reply by jspann93** [reply]
Okay no problem, thanks for the reply. I was mainly curious because sometimes when I Control+C the data copies just fine but most of the time it doesn't and I haven't been able to figure out why that is. Oh well, it's not a big deal. Thanks!
